### DISCUSSION TITLE
Configuration queries

### DISCUSSION THREAD

**Discussion by SleepyCatten** [RakambdaOrg/FallingTree#321] [Q&A] [answered]
Hi,

Sorry to trouble you. I have reviewed the Wiki, but I nonetheless remain a little confused over how to configure the mod in certain ways.

What I am seeking to do is:

• Have the mod on by default for all players.
• Have all trees allowed (including mushroom "trees" and nether "trees").
• Have the mod work without the need to sneak/crouch (i.e., it'll just work whenever any axe is used on a tree).

Sorry for being dense here 😣

**Comment by Rakambda** [comment, marked as answer]
From what I see seems like the default config will work fine for you.

> Have the mod on by default for all players

By default all players are able to use the mod. What would affect them are the tools you configure (by default only axes will work).

> Have all trees allowed (including mushroom "trees" and nether "trees")

This is already the case. Some other mods trees may not all be detected (depends if the mod author tags them as tree), so maybe you may need to add some more manually.

>  Have the mod work without the need to sneak/crouch (i.e., it'll just work whenever any axe is used on a tree).

This is the default.

**Comment by SleepyCatten** [comment]
Thanks for the reply!

Just to confirm, which is the precise option in the settings that makes the mod work irrespective of whether you're sneaking or not?

**Reply by Rakambda** [reply]
https://github.com/RakSrinaNa/FallingTree/wiki/Settings---General
https://github.com/RakSrinaNa/FallingTree/wiki/Settings---Examples

The one called `reverseSneaking`

**Comment by SleepyCatten** [comment]
Thank you kindly!

Sorry again if I'm being dense, but just that just make the mod work when you're not sneaking and stop it working when you are sneaking?

If so, I'm looking for a setting that enables the mod all the time (i.e., enabled if you are sneaking and also enabled if you aren't sneaking).

**Reply by Rakambda** [reply]
Currently there isn't that option, either one or the other will disable it.

This can be changed as an improvement (#322 )

**Comment by SleepyCatten** [comment]
Thanks 💟

**Reply by Rakambda** [reply]
Version 3.4.0+, option `sneakMode` set to `IGNORE` will enable the mod when both sneaking and not sneaking.
